engine/src/w32spec.cpp

```diff
@@ -98,10 +98,59 @@ void MCS_loadfile(MCExecPoint &ep, Boolean binary)
 #ifdef /* MCS_savefile_dsk_w32 */ LEGACY_SYSTEM
 void MCS_savefile(const MCString &fname, MCExecPoint &data, Boolean binary)
 {
-	if (!MCSecureModeCanAccessDisk())
+	char *t_path = NULL;
```

Comment:
Its a mis-merge isn't it? It appears to have pasted code from the resolution function into the legacy savefile function... Probably best to revert that part of the code back to what it should be as it shouldn't have changed.